Utilities/File.h

```diff
@@ -733,9 +733,18 @@ namespace fs
 		// Always use write flag, remove read flag
 		if (fs::file f{path, mode + fs::write - fs::read})
 		{
-			// Write args sequentially
-			(f.write(args), ...);
-			return true;
+			if constexpr ((std::is_pointer_v<Args> || ...))
```

Comment:
Is this specialization really better? It seems like it would do the exact same thing, but my knowledge of modern c++ templates and metas isn't as polished.
CC @Nekotekina

Reply:
Well, as a workaround it may be OK, although I think it's missing argument count check (sizeof... should equal 2)

Reply:
Fixed.